source/particles/particle_handler.cc

```diff
@@ -1033,7 +1033,7 @@ namespace Particles
 
           const unsigned int closest_vertex =
             GridTools::find_closest_vertex_of_cell<dim, spacedim>(
-              current_cell, out_particle->get_location());
+              current_cell, out_particle->get_location(), *mapping);
```

Comment:
I agree this looks like a bug, this accidental omission of the mapping parameter is happening quite often :disappointed: . But this instance also confuses me, this is only relevant for mappings that do not preserve vertex locations, right? So at least in the case of ASPECT it is only problematic when we use a MappingQEulerian for mesh deformation.

Reply:
I think that's correct.